server/motd.lua

```diff
@@ -0,0 +1,96 @@
+local acknowledged = GetConvar('qbx:acknowledge', 'false') == 'true'
+local messagesUrl = GetConvar('qbx:serviceMessagesUrl', 'https://raw.githubusercontent.com/Qbox-project/txAdminRecipe/refs/heads/main/service-messages.json')
+local resourceVersion = GetResourceMetadata(cache.resource, 'version', 0)
+
+---@param str string
+---@param prefix string
+---@return boolean
+local function startsWith(str, prefix)
+    return str:sub(1, prefix:len()) == prefix
```

Comment:
Should we handle the case where str is an empty string?

Reply:
Currently it will make the message never show up. Should we make it otherwise?

Reply:
That's probably fine. As long as it isn't an error condition.

Reply:
From my testing, no inputs should result in errors.